Code/LauncherUnified/Launcher.cpp

```diff
@@ -697,6 +699,11 @@ namespace O3DELauncher
             status = ReturnCode::ErrCrySystemInterface;
         }
 
+        if ((gEnv) && (gEnv->pSystem) && (gEnv->pSystem->GetILevelSystem()))
+        {
+            gEnv->pSystem->GetILevelSystem()->UnloadLevel();
+        }
```

Comment:
Why not have the LevelSystem destructor do this?

Reply:
+1 - ideally we don't want to be adding any additional dependencies on the level system from anywhere considering we're trying to get rid of it.

Reply:
Fixed.